### DISCUSSION TITLE
Longhorn 1.6.2 - pvc is not ready for workloads

### DISCUSSION THREAD

**Discussion by zohebk8s** [longhorn/longhorn#8641] [General]
Hi Team,

I’ve deployed longhorn using helm and all the pods are up and running and even longhorn UI is working fine.

However when I deploy statefulset, pvc attachment is failing with below error:

volume pvc-3673848 is not ready for workloads 

cluster is RKE2 and the version is v1.28.9+rke2r1

**Comment by zohebk8s** [comment]
@innobead Any suggestions on this?

Is it something specific to longhorn version?

**Reply by innobead** [reply]
Let me convert this to a discussion first, as this is a common function if it does not work, that could be an env issue or prerequisite. 

> volume pvc-3673848 is not ready for workloads

Have you seen any errors in the PVC? You can kubectl describe it to see any info there first. 

cc @DamiaSan (coordinator)

**Reply by zohebk8s** [reply]
@innobead Thanks for your response. I’ve managed to fix it with iscsi drivers.

I will deploy few more and test. Hopefully it will work.
